PSFzf.Git.ps1

```diff
@@ -39,6 +39,15 @@ function SetupGitPaths() {
             $gitInfo = Get-Command git.exe -ErrorAction SilentlyContinue
             $script:foundGit = $null -ne $gitInfo
             if ($script:foundGit) {
+                # Detect if scoop is installed
+                $script:scoopInfo = Get-Command scoop -ErrorAction SilentlyContinue
+                if ($null -ne $script:scoopInfo) {
+                    # Detect if git is installed using scoop (using shims)
+                    if ((Split-Path $gitInfo.Source -Parent) -eq (Split-Path (Get-Command scoop).Source -Parent)) {
```

Comment:
I think we can reuse `$script:scoopInfo` instead of calling `Get-Command` again. `Get-Command` can be quite slow.

Reply:
Yeah, definitely! I quickly made the changes and didn't have performance issues in mind. The new commit should fix that.